source/javascripts/main.js

```diff
@@ -7,6 +7,12 @@
 //= require_tree ./directives/
 //= require_tree ./controllers/
 
+// underscore template settings
 _.templateSettings = {
     interpolate: /\{\{(.+?)\}\}/g
 };
+
+// polyfill for IE
+if (!Number.MAX_SAFE_INTEGER) {
+    Number.MAX_SAFE_INTEGER = Math.pow(2, 53) - 1;
+}
```

Comment:
Based on this: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER IE needs polyfill

Reply:
hmm yeah a browser support doc would be useful. I'll leave it here just in case